sys/vm_map.c

```diff
@@ -139,7 +139,7 @@ void vm_map_protect(vm_map_t *map, vm_addr_t start, vm_addr_t end,
 
 int vm_map_findspace(vm_map_t *map, vm_addr_t start, size_t length,
```

Comment:
Could you add a comment that would describe in a few words what that function is supposed to do? Is `start` argument only a hint?